src/stackset-stack.ts

```diff
@@ -36,6 +36,14 @@ export class StackSetStackSynthesizer extends StackSynthesizer {
   }
 }
 
+/**
+ * Properties for a StackSet Deployment. Subset of normal Stack properties.
+ */
+interface StackSetStackProps {
```

Comment:
Can you add the doc strings for these properties? It's fine to just copy from the StackProps doc strings